server/fishtest/static/js/application.js

```diff
@@ -55,12 +55,13 @@ $(() => {
     let theme = $.cookie('theme') || 'light';
     $("#change-color-theme").click(function() {
       if (theme === 'light') {
+        const darkThemeSha256 = $("meta[name='dark-theme-sha256']").attr("content");
```

Comment:
Please delete these lines in the comment or use the same command eg `openssl theme.dark.css dgst -sha256 -binary | openssl base64 -A`

https://github.com/glinscott/fishtest/blob/4c0987966e0e9f09d7c73f632b2500de0d8114ba/server/fishtest/static/js/application.js#L53-L54